components/gitpod-db/src/typeorm/migration/1662027342962-WorkspaceInstanceUsageUpdate.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { columnExists } from "./helper/helper";
+
+const TABLE_NAME = "d_b_workspace_instance_usage";
+const COLUMN_NAME = "startedAt";
+
+export class WorkspaceInstanceUsageUpdate1662027342962 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        if (await columnExists(queryRunner, TABLE_NAME, COLUMN_NAME)) {
+            await queryRunner.query(
+                `ALTER TABLE ${TABLE_NAME} CHANGE COLUMN ${COLUMN_NAME} timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6), ALGORITHM=INPLACE, LOCK=NONE`,
```

Comment:
Should we actually set a default value here? Does it make any sense?

Reply:
No, we should leave it NULL if it is null.